src/eclingo/parsing/parser.py

```diff
@@ -108,3 +113,6 @@ def _parse_show_signature_statement(self, statement: ast.AST) -> List[ShowStatem
 
 def parse_program(program: str, callback: _CallbackType, parameters: Sequence[str] = (), name: str = "base", config: AppConfig = AppConfig(semantics="c19-1")) -> None:
     _ProgramParser(program, callback, parameters, name, config)()
+    
+def reify_parse_program(program: str, callback: _CallbackType, parameters: Sequence[str] = (), name: str = "base", config: AppConfig = AppConfig(semantics="c19-1", verbose=0, use_reification=True)) -> None:
```

Comment:
Why ```use_reification=True```? I was expecting ```False```.

Reply:
I declared use_reification=False here:
https://github.com/jorgefandinno/eclingo/blob/70341d54f1e7575ff7c8b576c8b1663003b7305c/src/eclingo/config.py#L6-L9
as a baseline for use reification, and then whenever we want to use it, we set it to True, but if it is the other way around I'll change it.

Reply:
you are right, I miss read the context.

It seems that this function is only referenced in the tests.

Is that correct?

Reply:
If you mean the function reify_parse_program, yes. It is only references in the tests. parse_program is used for test_parsing and because we wanted to introduced the reification option I thought to just split it into another one and use it for test_reification

Reply:
Yes, I refer to reify_parse_program(). It is not a good idea to have a function in the code only for testing purposes. This means that the thing we are testing is different than the thing we are running in production code.

Consider changing https://github.com/jorgefandinno/eclingo/blob/a6814248671e524ae3ab7cd1cc271b1d7b944940/tests/test_reification.py#L23

by 
```python
parser.parse_program(stm, ret.append, parameters, name, config = AppConfig(semantics="c19-1", verbose=0, use_reification=True))
```
Would that make sense?

Reply:
Yes, that makes sense. I thought on changing it on the parser.py file but then it was when I was getting problems with the test_parsing, but it works like this. I have changed it now.

On the other hand, I have updated more tests. I'll finish the rest later.

Just to make it clear, the convention for strong negation has changed from sn_u(a) to sn_a with some changes that you introduced a couple days ago right?

Reply:
> Just to make it clear, the convention for strong negation has changed from sn_u(a) to sn_a with some changes that you introduced a couple days ago right?

Strong negation should be represented as ```u(-a)``` when not in an epistemic literal. If in an epistemic literal, it doesn't really matter as this is our next step.

Reply:
> > Just to make it clear, the convention for strong negation has changed from sn_u(a) to sn_a with some changes that you introduced a couple days ago right?
> 
> Strong negation should be represented as `u(-a)` when not in an epistemic literal. If in an epistemic literal, it doesn't really matter as this is our next step.

You can comment out tests with negation in epistemic literals for now. Once we address, we add them back.